packages/nextjs/src/performance/client.ts

```diff
@@ -18,6 +17,9 @@ const global = getGlobalObject<Window>();
 
 type StartTransactionCb = (context: TransactionContext) => Transaction | undefined;
 
+declare const __INJECTED_ROUTE_TABLE__: { [parameterizedRoute: string]: string } | undefined;
+const routeTable = typeof __INJECTED_ROUTE_TABLE__ === 'undefined' ? undefined : __INJECTED_ROUTE_TABLE__;
```

Comment:
is it possible to just inject a subset of the routes in? And then lazy load the rest in?

Reply:
Went for a different approach using data the framework already provides.